jupyterhub_traefik_proxy/traefik_utils.py

```diff
@@ -9,11 +9,13 @@
 import toml
 
 from urllib.parse import urlparse
```

Comment:
when from the same module, we can import multiple things on one line:

```suggestion
from urllib.parse import unquote, urlparse
```